web/portal/client/src/components/Dashboard/cards/CardsAmountVpbx.tsx

```diff
@@ -1,10 +1,9 @@
 import _ from '@irontec/ivoz-ui/services/translations/translate';
 import { Link } from 'react-router-dom';
 
-import { DashboardData } from '../@types';
-import IconClientsDashboard from '../IconClients';
-import IconPlatformsDashboard from '../IconPlatforms';
-import IconUsersDashboard from '../IconUsers';
+import DialpadIcon from '../DialpadIcon';
```

Comment:
There are actually four client portals, based on loged in client type (vpbx, retail, residential and wholesale). This change must be applied to CardsAmountResidential.tsx, CardsAmountRetail.tsx and CardsAmountWholeSale.tsx as well. 

I guess that IconClients, IconPlatforms and IconUsers are no longer needed so remove them.